dynamic-property-api/src/main/java/ru/fix/dynamic/property/source/ConstantProperty.java

```diff
@@ -1,7 +1,10 @@
-package ru.fix.dynamic.property.api;
+package ru.fix.dynamic.property.source;
```

Comment:
Seems like ConstantProperty changed it's package from `api` to `source` but still sitting in old folder?  
Could you please check if this is not an mistake?

Reply:
move back